### ISSUE TITLE
custom POIs from csv file

### ISSUE THREAD

**Issue by yipcma** [hove-io/navitia#1882]
@antoine-de for documentation/example file of custom POIs in a csv file instead of/in addition to ingesting from .osm.pbf 

Thank you very much.

**Comment by yipcma** [comment]
Hi @pbougue , I'd like to follow up on how to add POIs/places so that navitia can use those locations as origins/destinations for routing. Thank you.

**Comment by pbougue** [comment]
**The OSM way**
For the osm POI here is the default value of params that handle it : https://github.com/CanalTP/navitia/blob/dev/source/ed/default_poi_types.h
The json describes what OSM tags are "detected" as POI and what is their final POI-type (french types).
If you want to change that json parameter, the easiest is probably to do that in tyr: https://github.com/CanalTP/navitia/tree/dev/source/tyr#poitypes
or you can provide it to osm2ed using "-p" then the json-as-a-string.
The default list of poi_types: https://api.navitia.io/v1/coverage/fr-idf/poi_types
A poi output when calling /places : https://api.navitia.io/v1/coverage/fr-idf/places?q=jardin%20espace%20vert

**The CSV way**
For the csv part (so no-OSM POI) you have to provide the poi files in a zip, and there seems to be no doc for that... Except for that short, but usefull warning to go back to osm POIs: 	https://github.com/CanalTP/navitia/tree/dev/source/ed#poi2ed
You have 3 csv files:
* poi_type.txt : poi_type_name;poi_type_id
* poi.txt : poi_id;poi_name;poi_weight;poi_visible;poi_lon;poi_lat;poi_type_id
* poi_properties.txt (probably not required): poi_id;key;value

**Comment by yipcma** [comment]
What exactly is the format for the osm files and steps to incorporate it? do i just drag and drop a .pbf file to the data folder where i put the gtfs.zip under a region folder?

**Comment by pbougue** [comment]
Yes, you put a regular osm.pbf file, and it should be processed both for street network and POIs.
